packages/fury-adapter-oas3-parser/lib/parser/oas/parseParameterObject.js

```diff
@@ -110,6 +110,8 @@ function parseParameterObject(context, object) {
 
   const createReservedHeaderNamesWarning = createWarning(namespace, `'${name}' 'name' in location 'header' should not be 'Accept', 'Content-Type' or 'Authorization'`);
 
+  const createMalformedUriCharacterError = createError(namespace, `'${name}' 'name' in location 'query' contains URI malformed cahracters`);
```

Comment:
small typo for characters:

```suggestion
  const createMalformedUriCharacterError = createError(namespace, `'${name}' 'name' in location 'query' contains URI malformed characters`);
```